packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1576,10 +1576,13 @@ class _HourMinuteTextFieldState extends State<_HourMinuteTextField> {
         child: TextFormField(
           expands: true,
           maxLines: null,
+          inputFormatters: <TextInputFormatter>[
```

Comment:
1 line?

Reply:
maxLines has to be null for the `expands` logic to work, so we limit to 2 characters